.github/workflows/review.yml

```diff
@@ -34,13 +34,47 @@ jobs:
   review:
     runs-on: ubuntu-latest
     needs: build
-    name: Review
+    name: Review + Acceptance Tests
```

Comment:
When are these test "acceptance test" or "smoke tests" or "e2e tests" or "integration tests"? Trying to understand how do we define those?

Reply:
Yes It would be better to have as less synonyms as possible. In this case acceptance tests are same as browser tests. Usually smoke tests are only small subselection of acceptance tests that verify that the core functionality is working in production.

Reply:
https://www.guru99.com/smoke-sanity-testing.html#:~:text=Smoke%20Testing%20verifies%20the%20critical,whereas%20Sanity%20testing%20isn't.

Reply:
Perhaps later when we have more browser tests, some of them are located in different smoke test suite (sub) folder and only them are runned against production. Perhaps we need another yarn script for smoke tests too then.